tools/design/logos/MNE-CPP_Logo.svg

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did the Logo Change? - Hard to review if this is okay?

Reply:
Nope. I did not change the logo. I did open it with Illustrator though. I think this should be fine.